File: string_middle.java
class StringExercise{
    String getMiddle(String str){
      int start = (int) (str.length()-1)/2;
      int end = Math.round((str.length())/2) + 1;
    	return str.substring(start, end);
    }
    // 아래는 테스트로 출력해 보기 위한 코드입니다.
    public static void  main(String[] args){
        StringExercise se = new StringExercise();
        System.out.println(se.getMiddle("test"));
    }
}

File: string_middle.py
def string_middle(str):
    # 함수를 완성하세요
    return str[(len(str)-1)//2 : round((len(str))/2)+1]

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print(string_middle("test"))
print(string_middle("power"))
